pkg/server/wire.go

```diff
@@ -180,6 +181,8 @@ var wireBasicSet = wire.NewSet(
 	wire.Bind(new(teamguardian.Store), new(*teamguardianDatabase.TeamGuardianStoreImpl)),
 	teamguardianManager.ProvideService,
 	wire.Bind(new(teamguardian.TeamGuardian), new(*teamguardianManager.Service)),
+	featuremgmt.ProvideManagerService,
+	featuremgmt.ProvideToggles,
```

Comment:
Perhaps over complicating this....  

Manager = can control (and eventually change) that flags
Toggles = readonly settings

Reply:
It's a reasonable distinction, I had been considering a follow-up PR for licensing to do something similar as the majority of consumers only need a very small subset of the current interface.